Resources/Prototypes/secret_weights.yml

```diff
@@ -1,8 +1,30 @@
 - type: weightedRandom
   id: Secret
-  weights:
+  weights: #these are the presets!!! NOT gamerules.
     Nukeops: 0.20
     Traitor: 0.50
     Zombie: 0.10
     Revolutionary: 0.20
-    
\ No newline at end of file
+
+- type: weightedRandom
+  id: Secret_MRP
+  weights:
+    Nukeops: 0.15
+    Traitor: 0.40
+    Zombie: 0.10
+    Revolutionary: 0.10
+    Extended: 0.25
```

Comment:
Extended is for admins to do events in. AKA this is a greenshift. This will need discussion most likely

Reply:
Recent admin vote desires the shittersafari gamemode to return for the purpose of catching rule breakers.